### ISSUE TITLE
Loss Function

### ISSUE THREAD

**Issue by SaharAlmahfouzNasser** [pallashadow/SuperGlue-pytorch#10]
Hi, Could you please guide me to the line in which you call the loss function?
In other words what loss did you use to train SuperGlue?

Thanks,
Sahar

**Comment by Shuhul24** [comment]
Did you get any idea about the loss function. Where is it involved in the code?
@pallashadow @SaharAlmahfouzNasser

**Comment by pallashadow** [comment]
https://github.com/pallashadow/SuperGlue-pytorch/blob/master/models/superglueLoss.py
@Shuhul24 @SaharAlmahfouzNasser
